web-services/client/src/main/java/datawave/webservice/model/ModelList.java

```diff
@@ -55,7 +61,8 @@ public String getPageHeader() {
      */
     @Override
     public String getHeadContent() {
-        return EMPTY;
+        String dataTablesHeader = MessageFormat.format(DATA_TABLES_TEMPLATE, "/", "/");
```

Comment:
That is a good idea.  I will give it a try.   I need to modify this pull request because I implemented search on the ModelList page and not the Model Page that you get to by clicking a link on the ModelList page.  That is what @ivakegg really wanted.  
 
By the way, the Quickstart uses an accumulo metadata table name called datawave.metadata which is not the default.  I need to pass the table name in on the query string to get the Model List page to render correctly I have to add the non-default tablename to the URLs for the Models that are generated in the Model List page too.

This PR needs is almost done but I am trying to resolve the merging conflict with Model.java.